README.md

```diff
@@ -11,8 +11,17 @@ A vulnerability scanner for container images and filesystems. [Easily install th
 
 **Features**
 - Scan the contents of a container image or filesystem to find known vulnerabilities.
-- Find vulnerabilities for major operating system packages across Alpine, BusyBox, CentOS / Red Hat, and Debian / Ubuntu flavored distributions
-- Find vulnerabilities for Ruby (Bundler), Java, JavaScript (npm/yarn), Python (Egg/Wheel) packages, and Python Pip/requirements.txt listings
+- Find vulnerabilities for major operating system packages
+  - Alpine, 
+  - BusyBox
+  - CentOS / Red Hat
+  - Debian / Ubuntu flavored distributions
```

Comment:
Are there Ubuntu flavored distros? I think there are a couple but usually people refer to Debian-based distros? (this is fine to ignore)

```suggestion
  - Debian, and Debian flavored distributions
```